.rubocop.yml

```diff
@@ -31,9 +31,6 @@ Style/BlockDelimiters:
   Description: Prefer braces for chaining. Mostly an aesthetical choice. Better to
     be consistent then.
   EnforcedStyle: braces_for_chaining
-Style/ClassAndModuleChildren:
```

Comment:
This module uses the new pdk. Therefore, if we want to remove this rule you will need to update this modules .sync.yml to disable this rule in rubocop.yml instead of removing it manually.  

When using the pdk locally run ‘pdk update’ and notice your change will get reverted.

If you need help with this let me know!